emby_client.py

```diff
@@ -24,7 +24,7 @@
 AUTH_PASSWORD_KEY = "Pw"
 
 # query param constants
-AUDIO_STREAM = "universal"
+AUDIO_STREAM = "universal.mp3"
```

Comment:
@fnorbenden,

I also had to make a slight change here to use universal.{container} as I was unable to get the FLAC files to transcode to MP3 without it.